src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -568,6 +569,10 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 				this.hasHadInput = true;
 			}
 
+			if (isIPad && event.key === 'c' && standardKeyboardEvent.ctrlKey) {
```

Comment:
This would be better put into VS Code's keybinding system, rather than inline in the terminal's key processing. So a special `isIPad` keybinding here: https://github.com/Microsoft/vscode/blob/3021226eefdfc149c0b7b7f3a4a162524ada01e6/src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts#L116-L116

That way it will show up in keybindings.json, can be removed, is more declarative, etc.